bridge/bridgeapp/forms.py

```diff
@@ -10,24 +10,27 @@ class ResponseForm(forms.Form):
         required=True,
         label='',
         widget=forms.Textarea(attrs={
-            'rows': '2',
+            'rows': '3',
```

Comment:
I believe the number of rows or just how the input field will appear is a determination belonging to the styling stage of the app. Anything done now is subject to change once the style are discussed, so why bother now?